### ISSUE TITLE
confusing interrupt naming when using gvnic

### ISSUE THREAD

**Issue by romange** [GoogleCloudPlatform/compute-virtual-ethernet-linux#30]
I am using gvnic driver integrated with the kernel (v1.0).

`/proc/interrupts` has the following entries related to gvnic:
`eth%d-ntfy-block.0 ,  eth%d-ntfy-block.1, ... eth%d-ntfy-block.15`

is this `%d` by design? seems like a formatting bug. Also how can one know which interrupts map to each queue? 
my instance (n2d-standard-16) has 8 rx queues and 8 tx queues

**Comment by davidturchak** [comment]
Take a look here: 
https://lore.kernel.org/netdev/20230203212045.1298677-1-pkaligineedi@google.com/T/

**Comment by jfraker** [comment]
Thanks for the report!

This should be fixed as of the v1.4.0
```
 24:         65          0   PCI-MSI 65536-edge      gve-ntfy-blk0@pci:0000:00:04.0
 25:         71          1   PCI-MSI 65537-edge      gve-ntfy-blk1@pci:0000:00:04.0
 26:          0          0   PCI-MSI 65538-edge      gve-mgmnt@pci:0000:00:04.0
 ```
